pkg/cmd/pulumi/packagecmd/package_publish_sdk.go

```diff
@@ -64,6 +66,39 @@ func newPackagePublishSdkCmd() *cobra.Command {
 	return cmd
 }
 
+// determineNPMTagForStableVersion determines if we should tag this version as a backport, rather than as latest.
+func determineNPMTagForStableVersion(npm, pkgName, currentVersion string) (string, error) {
+	// Parse the current version first
+	currentVer, err := semver.ParseTolerant(strings.TrimSpace(currentVersion))
```

Comment:
`strings.TrimSpace` is redundant here, semver.ParseTolerant already deals with that correctly.